docs/Installation/Installation.md

```diff
@@ -258,6 +260,10 @@ To build SpECTRE with Singularity you must:
 - You should edit source files in SPECTRE_ROOT in a separate terminal
   outside the container, and use the container only for compiling and running
   the code.
+- We force the use of python2 here because the Singularity container uses
+  python3.8 by default. However, unless you have python3.8 in your environment
+  outside the container, this will create problems with git hooks, thus it is
+  suggested to use cmake with python2 inside Singularity.
```

Comment:
Some people won't have python 2 these days.  Distros are starting to drop it.

Reply:
So then do we just leave it up to the user to figure out their python dependencies?

Reply:
Pretty much. Most people should be using python3 nowadays, but it's annoying that python decides that any minor version change means they can change anything and everything in the language. Python stability basically disappeared in 3.x

Reply:
Ok instead of changing it in the example cmake command, I'll keep a note saying users should be aware of this.

Reply:
Yep, sounds reasonable to me :)